src/main/java/org/opentripplanner/framework/concurrent/OtpRequestThreadFactory.java

```diff
@@ -0,0 +1,28 @@
+package org.opentripplanner.framework.concurrent;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import java.util.concurrent.ThreadFactory;
+import javax.annotation.Nonnull;
+
+/**
+ * This thread pool factory should be used to create all threads handling user requests in OTP. It
+ * allows OTP to decorate threads and tasks. It should not be used for updater threads.
```

Comment:
Can you explain why it should not be used by updater threads? It's no obvious to me.